modtools/create-unit.lua

```diff
@@ -579,11 +601,8 @@ if args.nick and type(args.nick) == 'string' then
   dfhack.units.setNickname(df.unit.find(unitId), args.nick)
 end
 
-if civ_id then
-  local u = df.unit.find(unitId)
-  u.civ_id = civ_id
-end
 
+--[[ depracated by amostubal.  no longer necessary as we move the cursor in the creation of the unit.  no longer needs to utilize teleport...
 if args.location then
```

Comment:
yes we can, I really try not to delete whole lines sections, that much.   The location is just passed along to the actual function that creates the unit.